### ISSUE TITLE
Not able to see the graphs on the katib UI

### ISSUE THREAD

**Issue by kruthikakr** [kubeflow/katib#1607]
/kind bug

I am trying to run the example kubectl apply -f pytorchjob-example.yaml .
The experiment runs forever , i am not getting the graph / progress of the trial in the katib UI .. 

kubectl get trials -n kubeflow
NAME                          TYPE      STATUS   AGE
pytorchjob-example-4skcff8k   Running   True     85m
pytorchjob-example-8kd47ghq   Running   True     85m
pytorchjob-example-dbqcr6zf   Running   True     85m

kubectl get trials pytorchjob-example-dbqcr6zf -o yaml -n kubeflow
apiVersion: kubeflow.org/v1beta1
kind: Trial
metadata:
  creationTimestamp: "2021-08-09T08:17:57Z"
  finalizers:
  - clean-metrics-in-db
  generation: 1
  labels:
    experiment: pytorchjob-example
  name: pytorchjob-example-dbqcr6zf
  namespace: kubeflow
  ownerReferences:
  - apiVersion: kubeflow.org/v1beta1
    blockOwnerDeletion: true
    controller: true
    kind: Experiment
    name: pytorchjob-example
    uid: 68875f04-ef0b-41ce-9d0a-2e21e9f8d647
  resourceVersion: "5583217"
  uid: ef9e8e1f-7712-4631-9742-4a8e23b28128
spec:
  metricsCollector: {}
  objective:
    goal: 0.001
    objectiveMetricName: loss
    type: minimize
  parameterAssignments:
  - name: lr
    value: "0.03177173624863413"
  - name: momentum
    value: "0.8724378457198758"
  primaryContainerName: pytorch
  runSpec:
    apiVersion: kubeflow.org/v1
    kind: PyTorchJob
    metadata:
      name: pytorchjob-example-dbqcr6zf
      namespace: kubeflow
    spec:
      pytorchReplicaSpecs:
        Master:
          replicas: 1
          restartPolicy: OnFailure
          template:
            spec:
              containers:
              - command:
                - python3
                - /opt/pytorch-mnist/mnist.py
                - --epochs=1
                - --lr=0.03177173624863413
                - --momentum=0.8724378457198758
                image: docker.io/kubeflowkatib/pytorch-mnist:v1beta1-45c5727
                name: pytorch
        Worker:
          replicas: 2
          restartPolicy: OnFailure
          template:
            spec:
              containers:
              - command:
                - python3
                - /opt/pytorch-mnist/mnist.py
                - --epochs=1
                - --lr=0.03177173624863413
                - --momentum=0.8724378457198758
                image: docker.io/kubeflowkatib/pytorch-mnist:v1beta1-45c5727
                name: pytorch
status:
  conditions:
  - lastTransitionTime: "2021-08-09T08:17:57Z"
    lastUpdateTime: "2021-08-09T08:17:57Z"
    message: Trial is created
    reason: TrialCreated
    status: "True"
    type: Created
  - lastTransitionTime: "2021-08-09T08:17:57Z"
    lastUpdateTime: "2021-08-09T08:17:57Z"
    message: Trial is running
    reason: TrialRunning
    status: "True"
    type: Running
  startTime: "2021-08-09T08:17:57Z"

**What did you expect to happen:**

I expect to see the progress the job in the UI with experiment graphs , also some error message if there is any problem. 

**Environment:**

- Kubeflow version (`kfctl version`): kfctl v1.2.0-0-gbc038f9
- Minikube version (`minikube version`): azure platform
- Kubernetes version: (use `kubectl version`):v1.21.3

**Comment by andreyvelich** [comment]
@kruthikakr If you deploy Katib as part of Kubeflow you should create your Katib Experiments in User's Profile namespace.
Check guide here: https://www.kubeflow.org/docs/components/katib/hyperparameter/#pytorch-example
